test/integration/utils/sn-table.ts

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable class-methods-use-this */
+/* eslint-disable no-await-in-loop */
+import { Locator, Page } from '@playwright/test';
+import { Collection, Key } from '../types';
+
+type KeyPress = {
+  key: Key;
+  times: number;
+};
+
+const selectors: Collection = {
+  chart: '#chart-container',
+  text: (text: string): string => `text=${text}`,
+  focused: 'th[tabindex="0"], td[tabindex="0"]',
+};
+
+const textToDelete: string = '\nPress space to sort on this column';
+
+class SnTable {
```

Comment:
I would say that this is a [Page Object Model](https://playwright.dev/docs/pom) and should not be in the `utils` folder but rather a "Page Object Model" folder?